ethereum/networks/src/main/java/tech/pegasys/teku/networks/Eth2NetworkConfiguration.java

```diff
@@ -506,6 +509,18 @@ public Builder applyRopstenNetworkDefaults() {
               "enr:-Iq4QMCTfIMXnow27baRUb35Q8iiFHSIDBJh6hQM5Axohhf4b6Kr_cOCu0htQ5WvVqKvFgY28893DHAg8gnBAXsAVqmGAX53x8JggmlkgnY0gmlwhLKAlv6Jc2VjcDI1NmsxoQK6S-Cii_KmfFdUJL2TANL3ksaKUnNXvTCv1tLwXs0QgIN1ZHCCIyk");
     }
 
+    private Builder applySepoliaNetworkDefaults() {
+      return applyTestnetDefaults()
+          .constants(SEPOLIA.configName())
+          .startupTimeoutSeconds(120)
+          .eth1DepositContractDeployBlock(1273020)
+          .defaultInitialState(
+              "https://github.com/eth-clients/merge-testnets/raw/9c873ab67b902aa676370a549129e5e91013afa3/sepolia/genesis.ssz")
+          .discoveryBootnodes(
+              // EF bootnode
+              "enr:-Iq4QMCTfIMXnow27baRUb35Q8iiFHSIDBJh6hQM5Axohhf4b6Kr_cOCu0htQ5WvVqKvFgY28893DHAg8gnBAXsAVqmGAX53x8JggmlkgnY0gmlwhLKAlv6Jc2VjcDI1NmsxoQK6S-Cii_KmfFdUJL2TANL3ksaKUnNXvTCv1tLwXs0QgIN1ZHCCIyk");
```

Comment:
Suspiciously this is the same as the ropsten bootnode.... It is the same as the one in the sepolia config info and will work, so let's go with it, but I pinged Pari on discord to double check.